File: ch01/ch01_06.py
from ch01_05 import letter_ngram
X = set(letter_ngram('paraparaparadise', 2))
Y = set(letter_ngram('paragraph', 2))
print(X | Y)
print(X & Y)
print(X - Y)
print('se' in X)
print('se' in Y)

File: ch01/ch01_06.rb
require 'set'
require_relative './ch01_05'

X = Set.new(letter_ngram('paraparaparadise', 2))
Y = Set.new(letter_ngram('paragraph', 2))
p X + Y
p X & Y
p X - Y
puts X.include?('se')
puts Y.include?('se')
